diff --git a/third_party/xla/third_party/farmhash/farmhash.BUILD b/third_party/xla/third_party/farmhash/farmhash.BUILD
--- a/third_party/xla/third_party/farmhash/farmhash.BUILD
+++ b/third_party/xla/third_party/farmhash/farmhash.BUILD
@@ -14,5 +14,5 @@
     # Disable __builtin_expect support on Windows
     copts = select({
-        ":windows": ["/DFARMHASH_OPTIONAL_BUILTIN_EXPECT"],
+        ":windows": ["-DFARMHASH_OPTIONAL_BUILTIN_EXPECT"],
         "//conditions:default": [],
     }),




diff --git a/tensorflow/compiler/mlir/lite/build_def.bzl b/tensorflow/compiler/mlir/lite/build_def.bzl
--- a/tensorflow/compiler/mlir/lite/build_def.bzl
+++ b/tensorflow/compiler/mlir/lite/build_def.bzl
@@ -43,4 +43,4 @@
             # copybara:uncomment_end_and_comment_begin
-            "/DTFL_COMPILE_LIBRARY",
-            "/wd4018",  # -Wno-sign-compare
+            "-DTFL_COMPILE_LIBRARY",
+            "-Wno-sign-compare",  # -Wno-sign-compare
             # copybara:comment_end
